README.md

```diff
@@ -40,7 +40,7 @@ These dependencies are subject to change so be sure to verify your version of My
 
 ### Install Steps
 
-Consult the [LORIS Wiki](https://github.com/aces/Loris/wiki/Setup) page on this [Install process](https://github.com/aces/Loris/wiki/Installing-Loris) for more information.
+Consult the [LORIS Installation Markdowns](/docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install) for more information.
```

Comment:
Same here: Absolute links will break readthedocs, and readthedocs can't manage links to folders.